src/main/java/jR_Project3/builders/users/SimpleUserBuilder.java

```diff
@@ -0,0 +1,28 @@
+package jR_Project3.builders.users;
+
+import jR_Project3.interfaces.UserBuilder;
+import jR_Project3.models.users.SimpleUser;
+
+public class SimpleUserBuilder implements UserBuilder {
```

Comment:
why just don't use @Builder annotation instead of writing own builder

Reply:
I wanted to write this myself for the first time :)